sdk/src/utils/request.ts

```diff
@@ -20,10 +21,8 @@ export const request = async <Output = any>(
 
 	const response = await fetcher(url, options);
 
-	const data = await extractData(response).catch((reason) => {
+	return await extractData(response).catch((reason) => {
```

Comment:
I think we can omit the `await` now
```suggestion
	return extractData(response).catch((reason) => {
```